### DISCUSSION TITLE
First person camera

### DISCUSSION THREAD

**Discussion by MRcreeper176** [efroemling/ballistica#743] [Ideas]
I think bombsquad will be even very more fun if someone code this plugin🤣🤣
I saw the 'manual camera' plugin that gives you access to change the camera position, but in this plugin the camera will be in front of the face of our charecter and it will move automatically as the charecter move.
If someone code this plugin, i think bombsquad can be the best game to play with VR!😂

**Comment by JoseANG3L** [comment]
They have already created that but the camera is very rough, a serious problem is the controls that only move in the global position of the map and not where you are looking,  although it would be a good idea to implement it in some minigames.

**Reply by MRcreeper176** [reply]
Yes, the hardest part is the controllers; they don't sync with the camera.
Maybe @efroemling have some ideas about it😁.

**Reply by JoseANG3L** [reply]
There is a way to fix this, we have the front and back positions of the character, it could be calculated with some math.

**Reply by MRcreeper176** [reply]
the camera position will be updated overtime due to the movement of our character

**Reply by MRcreeper176** [reply]
*from the first persion camera, i mean we see the game from our character eyes, not from its back.

**Comment by MatiasMGz** [comment]
Any updates on this? It would be funny.
